.github/workflows/baseline.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     name: "Release Build (optimized)"
     runs-on: [self-hosted, linux, x64, general]
     steps:
-      - run: bash scripts/environment/bootstrap-self-hosted.sh
+      - uses: colpal/actions-clean@v1
```

Comment:
Not taking the suggestion to run it at the end?

Reply:
I considered it, but felt that doing it at the start was more fool-proof. That way, if a given job misses running it at the end, other jobs are unaffected.

Reply:
We could run it in _both_ places 🤔

Reply:
```yaml
  - uses: colpal/actions-clean@v1
    if: ${{ always() }} # To ensure this step runs even when earlier steps fail
```

This _seems_ fine, I'd prefer to not add more overhead to our CI time, it's already long enough

Reply:
Yeah, the issue I was trying to avoid was some job neglecting to run it. The way I have it now, only the job that forgot to run it would be affected rather than it affecting other jobs.

Reply:
OK, whatever works!